### PR TITLE
Minor v8 Fixes 5

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
- add erase blend mode to WebGPU
- set name correctly in renderer
- fix compatibility error if one filter in array is not compatible
- add default values to a uniform based on the type

### Summary
🎨🛠️🧪

This pull request adds or improves several features and utilities related to the rendering system of PixiJS. It introduces a new `erase` blend mode for GPU rendering, a new `name` property for renderer instances, and a new `getTestContext` function for shader program inspection. It also simplifies and enhances the `UniformGroup` class and its default value handling, and adds unit tests for it. It modifies the `FilterSystem` to check the filter compatibility with the renderer type. It adds a comment with tasks or issues for the mesh rendering module.

> _Oh we're the crew of the PixiJS ship_
> _And we sail the web with skill and wit_
> _We set the `blendMode` and the `UniformGroup`_
> _And we `erase` the pixels that we don't need_

### Walkthrough
*  Add `getTestContext` utility function to create and cache WebGL contexts for shader inspection ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-9ec62d7063d1bcbeaa0e9aebc8b8171c3736f90d3e3f04f7fb10ac5d186eb9b7R1-R47))
*  Modify `FilterSystem` logic to check filter compatibility with renderer type and skip incompatible filters ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-8040a574f56ede509ce32aed02c625d84c94550856acfeff39cc89bd4a80c256L188-R194))
*  Add `erase` blend mode to `GpuBlendModesToPixi` and `blendModeIds` objects for erasing pixels using destination alpha and color ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-716bcdff5c88fa4c3bebd661e5778d4ebd420bcbe80a593785703321a79d1391R123-R135), [link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-b37b12db1d38c9015ec877b1e025b5f4c4f15b6c5e220a8220022d30618f0e18R10))
*  Add `name` property to `AbstractRenderer` class to store and identify renderer instance name ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-27fa29afe053bf0bbdb6c29016c4da0abb00506b5b31c3e689aa84f35867aa42R73))
*  Add `defaultUniformValue` utility function to assign default values to uniforms based on type and size ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-647b6ac5b92f6bdbe79069bc3b40ca95adc0fb1b3f053888f16ad3d32a3c6d7cR1-R43))
*  Use `defaultUniformValue` function and nullish coalescing operator in `UniformGroup` class to simplify uniform assignment logic ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-984b1c73264c64684bfdb732d9172f4ea5e637483fdc6d6d45e313e7e08f52b7R2), [link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-984b1c73264c64684bfdb732d9172f4ea5e637483fdc6d6d45e313e7e08f52b7L63-R66))
*  Add `UniformGroup.test.ts` file with two unit tests for `UniformGroup` class using Jest framework ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-9bef18bd6b9b69b3863da5f526acc826cafd92a60522a9dd59347d8ade3615c2R1-R34))
*  Add comment with tasks or issues to migrate or resolve in mesh rendering module ([link](https://github.com/pixijs/pixijs/pull/9578/files?diff=unified&w=0#diff-3bdfdc4adda92f4e5a8ed7952bf8b57d81f129a0aee529f2accf8dcda4075a1cR1-R24))

### CODE REVIEW

File: src/renderers/gl/utils/getTestContext.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// import { ENV } from '@pixi/constants';
+// import { settings } from '@pixi/settings';
```
Comment (Zyie) [inline src/renderers/gl/utils/getTestContext.ts:2]: Should this file be deleted?

File: src/rendering/mesh/shared/sword-puzlle checks [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// migrate:
+// math-extra
```
Comment (Zyie) [inline src/rendering/mesh/shared/sword-puzlle checks:2]: This shouldn't be committed

File: src/rendering/scene/utils/buildInstructions.ts [hunk 3]
```diff
@@ -26,10 +26,11 @@ export function buildInstructions(layerGroup: LayerGroup, renderPipes: RenderPip
 
     collectAllRenderablesAdvanced(root, instructionSet, renderPipes, true);
 
-    // instructionSet.log();
     // TODO add some events / runners for build end
     renderPipes.batch.buildEnd(instructionSet);
     renderPipes.blendMode.buildEnd(instructionSet);
+
+    // instructionSet.log();
```
Comment (Zyie) [inline src/rendering/scene/utils/buildInstructions.ts:33]: Can be removed?

**Review by Zyie — APPROVED** [review]

File: src/rendering/mesh/shared/sword-puzlle checks (hunk 2 above)
Comment (GoodBoyDigital) [reply src/rendering/mesh/shared/sword-puzlle checks:2]: oops! will remove

**Pull request merged**